app/services/subscriptions/dates/yearly_service.rb

```diff
@@ -100,6 +100,13 @@ def compute_charges_duration(from_date:)
 
         compute_duration(from_date:)
       end
+
+      def period_started_in_last_year?(date)
+        return true if date.month < subscription_at.month
+        return true if (date.month == subscription_at.month) && (subscription_at.day > date.day)
```

Comment:
```suggestion
        return true if (date.month == subscription_at.month) && (date.day < subscription_at.day)
```

Looks good! 
Here's a small suggestion so that the line reads the same way as the one above